index.html

```diff
@@ -0,0 +1,214 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <h1 class="header">freeCodeCamp Survey Form</h1>
+    <p class="para-description">
+      Thank you for taking the time to help us improve the platform
+    </p>
+    <form>
+      <div class="form-container">
+        <label id="name" for="name">Name</label>
+        <input
+          type="text"
+          name="name"
+          id="input-name"
+          class="form-input"
+          placeholder="Enter your name"
+        />
+      </div>
+      <div class="form-container">
+        <label id="name" for="name">Email</label>
+        <input
+          type="text"
+          name="name"
+          id="input-name"
+          class="form-input"
+          placeholder="Enter your email"
+        />
+      </div>
```

Comment:
Please, use the correct type for the email input. The type should be "type = email" and not text as you have written. This helps in avoiding validation errors.

Reply:
- [ ] Nicely done 👍